nasher.nimble

```diff
@@ -1,6 +1,6 @@
 # Package
 
-version       = "0.14.2"
+version       = "0.14.7"
```

Comment:
This should be removed until a new tagged version is released.

Reply:
Yup, sorry, I do that so I can be sure which test version I'm running locally compared to my globally installed version.

Reply:
No problem. What I tend to do is `nasher build` and then symlink the locally built one to `nasher-dev` so I can test current versus testing version. There ought to be an easier way to manage multiple versions.